src/webapi/Service/AuthenticationConfigurationService.cs

```diff
@@ -0,0 +1,44 @@
+using EdFi.OdsApi.Sdk.Client;
+using EdFi.OdsApi.SdkClient;
+
+namespace eppeta.webapi.Service
+{
+    public class AuthenticationConfigurationService : IAuthenticationConfigurationService
+    {
+        private readonly string oauthUrl;
+        private readonly string clientKey;
+        private readonly string clientSecret;
+
+        public AuthenticationConfigurationService(string oauthUrl, string clientKey, string clientSecret)
+        {
+            this.oauthUrl = oauthUrl;
+            this.clientKey = clientKey;
+            this.clientSecret = clientSecret;
+        }
+
+        public Configuration GetAuthenticatedConfiguration()
+        {
+            // Trust all SSL certs -- needed unless signed SSL certificates are configured.
+            System.Net.ServicePointManager.ServerCertificateValidationCallback =
+                ((sender, certificate, chain, sslPolicyErrors) => true);
```

Comment:
This is option needs to be controlled by configuration - not good to always accept the SSL certificates. Need to add something to the appsettings file and the `AppSettings` singleton.